### PR TITLE
acala/karura: update testnet environments

### PR DESCRIPTION
**Pull request by andreclaro**
acala/karura: update testnet environments

Related with https://github.com/AcalaNetwork/Acala/discussions/2769#discussioncomment-9844505

### CODE REVIEW

**Review by evan-gray — DISMISSED** [review]

**Comment by bruce-riley** [comment]
Please update the addresses in the following places:

- https://github.com/wormhole-foundation/wormhole/blob/fdbfc0b085d71c84d104b55644604b5cc0254ec3/sdk/js/src/utils/consts.ts#L491-L500
- https://github.com/wormhole-foundation/wormhole/blob/fdbfc0b085d71c84d104b55644604b5cc0254ec3/sdk/testnet_consts.go#L23-L24

Remove NFT addresses here: https://github.com/wormhole-foundation/wormhole/blob/fdbfc0b085d71c84d104b55644604b5cc0254ec3/sdk/testnet_consts.go#L62-L63

Also need to generate new token bridge registration VAAs and update here: https://github.com/wormhole-foundation/wormhole/blob/fdbfc0b085d71c84d104b55644604b5cc0254ec3/deployments/testnet/tokenBridgeVAAs.csv#L12-L13

May want to do this in a follow on PR.

**Review by bruce-riley — DISMISSED** [review]

**Comment by andreclaro** [comment]
Related PR: https://github.com/wormhole-foundation/example-token-bridge-ui/pull/69/files

File: ethereum/env/.env.acala.testnet [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Acala testnet env
 # Rename to .env to use with truffle migrations
 
-# Gas price generated with ./sh/acala-gas-prices.sh
-FORGE_ARGS="--legacy --with-gas-price 213314634730"
+RPC_URL=https://eth-rpc-acala-testnet.aca-staging.network
+# Note: --with-gas-price is no longer required for testnet
```
Comment (zjb0807) [inline ethereum/env/.env.acala.testnet:5]: both testnet and mainnet don't need this args.  And the `./sh/acala-gas-prices.sh` / `./sh/karura-gas-prices.sh` shell can be removed.

**Review by evan-gray — APPROVED** [review]

**Comment by bruce-riley** [comment]
My comments about the const files were addressed in PR #4014.

**Review by bruce-riley — APPROVED** [review]

**Pull request merged**